src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/IHtmlHelper.cs

```diff
@@ -371,20 +370,6 @@ public interface IHtmlHelper
         string GenerateIdFromName(string fullName);
 
         /// <summary>
-        /// Returns information about about client validation rules for the specified <paramref name="metadata"/> or
-        /// <paramref name="expression"/>. Intended for use in <see cref="IHtmlHelper"/> extension methods.
-        /// </summary>
-        /// <param name="metadata">Metadata about the <see cref="object"/> of interest.</param>
-        /// <param name="expression">
-        /// Expression name, relative to the current model. Used to determine <see cref="ModelMetadata"/> when
-        /// <paramref name="metadata"/> is <c>null</c>; ignored otherwise.
-        /// </param>
-        /// <returns>An <see cref="IEnumerable{ModelClientValidationRule}"/> containing the relevant rules.</returns>
-        IEnumerable<ModelClientValidationRule> GetClientValidationRules(
```

Comment:
This method was in MVC 5 and we have fielded MVC 6 questions related to its use.